satis-config/govcms-stable.json

```diff
@@ -10,7 +10,7 @@
     ],
     "require": {
         "govcms/govcms": "1.3.0",
-        "govcms/scaffold-tooling": "2.0.0",
+        "govcms/scaffold-tooling": "1.1.0|2.0.1",
```

Comment:
The only really relevant line in this PR - we need to support both the 1.1.0 and 2.0.1 release. 

2.0.1 adds Solr 7.x support, which not all PaaS codebases are ready for.